.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.DS_Store
```

Comment:
to avoid the weird red error sign in github add a new line at the end of the file. VS code should have a setting to add a new line at the end of a file on save